src/valhalla_associate_segments.cc

```diff
@@ -196,10 +196,10 @@ struct edge_association {
   std::shared_ptr<vj::GraphTileBuilder> m_tile_builder;
 
   // Statistics
-  uint32_t success_count_;
-  uint32_t failure_count_;
-  uint32_t walk_count_;
-  uint32_t path_count_;
+  std::unordered_map<uint32_t, uint32_t> success_count_;
+  std::unordered_map<uint32_t, uint32_t> failure_count_;
+  std::unordered_map<uint32_t, uint32_t> walk_count_;
+  std::unordered_map<uint32_t, uint32_t> path_count_;
```

Comment:
guess this is OK - could just be an array (or vector) since the size is known.

Reply:
i assumed we would add level in the future  :-)